File: FORUM/Models/Reaction.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace FORUM.Models
{
    public class Reaction
    {

         [Key]
         public int Id { get; set; }

         public bool like { get; set; }

         public string date { get; set; }
         public Post post { get; set; }
        
         public User user { get; set; }

         public int postId { get; set; }
        public int  userId { get; set; }


    }
}

File: forum-angular/src/app/DTO/Reaction.ts
import { Post } from './Post';
import { Comment } from './Comment';
import { Category } from './Category';
import { User } from './User';
export class Reaction
{

  public  Id :number

  public  like :boolean

  public  date:string
  public  post:Post

  public  user:User

  public  postId:number
 public   userId:number
     constructor(){

     }
    //  public List<Comment> comments { get; set; }




}
